src/analysis/subspaces/nullspace/getNullSpace.m

```diff
@@ -28,13 +28,26 @@
 end
 
 gmscale=1;%by default, use geometric mean scaling of S
+archstr = computer('arch');
+archstr = lower(archstr);
+switch archstr
+    case {'win32', 'win64'}        
+        if issparse(S) && any(size(S) > 3000)

```

Comment:
Shouldn't there be an error/warning for models with `~issparse(S && any(size(S) > 3000)`?

Reply:
No. 
They will still be computed with matlabs internal methods.
Its just that for smaller models, I would expect the matlab internal functions to still perform reasonably well (while being more precise), and only on larger models approximations should be done (and LU is essentially an approximation).

Reply:
Just for my understanding: where are the smaller models computed with MATLAB internal functions?

Reply:
In the else clause.
```matlab
        if issparse(S) && any(size(S) > 3000)
            [Z,rankS] = sparseNull(S);
            
        else % for small matrices or those which are full, use the normal matlab functions
            Z = null(full(S));
            rankS = rank(full(S));
end
```

Reply:
I still think that there should be a warning for large models ...

Reply:
Well...
We don't have it for large models under linux (which use essentially the same methodology only with an external implementation that is faster because its C based), so I don't think its necessary for windows.

Reply:
Yes, under UNIX, there is no need for a warning as we have the compiled language implementation.  But on Windows, I consider it being a valuable warning ...